Bottom: 3bd8028a026e4eea5eed1f2ac52682172df6c6b2
Top:    a397cc95835b6afc66c370b430ec61ebb4d3100d
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2012-12-02 12:53:15 +0000

Refresh of push-special-case-thindraw-vs-

---

diff --git a/src/hid_draw.h b/src/hid_draw.h
index ef34be9..a6f8938 100644
--- a/src/hid_draw.h
+++ b/src/hid_draw.h
@@ -55,6 +55,7 @@ struct hid_draw_st
   void (*draw_pcb_arc) (hidGC gc, ArcType *arc);
   void (*draw_pcb_text) (hidGC gc, TextType *, Coord);
   void (*draw_pcb_polygon) (hidGC gc, PolygonType *poly, const BoxType *clip_box);
+
   void (*fill_pcb_pad) (hidGC gc, PadType *pad, bool clip, bool mask);
   void (*thindraw_pcb_pad) (hidGC gc, PadType *pad, bool clip, bool mask);
   void (*fill_pcb_pv) (hidGC fg_gc, hidGC bg_gc, PinType *pv, bool drawHole, bool mask);
